bin/oref0-setup.sh

```diff
@@ -452,7 +452,7 @@ else
     openaps device add pump mmeowlink subg_rfspy $ttyport $serial $radio_locale || die "Can't add pump"
     openaps alias add wait-for-silence '! bash -c "(mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 1 | grep -q comms && echo -n Radio ok, || openaps mmtune) && echo -n \" Listening: \"; for i in $(seq 1 100); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 30 2>/dev/null | egrep -v subg | egrep No && break; done"'
     openaps alias add wait-for-long-silence '! bash -c "echo -n \"Listening: \"; for i in $(seq 1 200); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 45 2>/dev/null | egrep -v subg | egrep No && break; done"'
-    if [[ ${radio_locale,,} =~ "WW" ]]; then
+    if [[ "$radio_locale" =~ "WW" ]]; then
```

Comment:
there is also some code for upcasing it in the script 
```
  # Force uppercase, just in case the user entered ww
      radio_locale=${radio_locale^^}
```
but only for user input. Should we always upcase or aways lowercase.  What do you suggest?